boards/arm/gd32a503v_eval/gd32f450z_eval.yaml

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2022 YuLong Yao <feilongphone@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+identifier: gd32a503v_eval
+name: GigaDevice GD32A503V-EVAL
+type: mcu
+arch: arm
+ram: 48
+flash: 384
+toolchain:
+  - zephyr
+  - gnuarmemb
+  - xtools
+supported:
```

Comment:
lost `adc`?

Reply:
Hi @feilongfl , there have two CI failure about adc test. Looks gd32 adc currently not support it. Please remove `adc` from here. Sorry for giving the incorrect suggestion.